src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components/macro'
+
```

Comment:
I suggest you use PropTypes!

Reply:
Till doesn't pass in props so prop types are not needed here